templates/rc.local.erb

```diff
@@ -25,16 +25,13 @@ echo \
 
 <%= @login_message %>
 
-
 For the best experience please log in via SSH
 
     IP: $IP_ADDRESS
     username: root
     password: $PASSWORD
 
-Your password was randomly generated from international place names
-It is stored in /var/local/password and will continue to appear on this screen
-
+The password can be found in /var/local/password as well as this screen.
```

Comment:
I'd rather it said something about being randomly generated, even if it were just a couple words.